api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -290,27 +290,9 @@ public DbUser createServiceAccountUser(String username) {
   }
 
   @Override
-  public DbUser createUser(
-      String givenName,
-      String familyName,
-      String userName,
-      String contactEmail,
-      String currentPosition,
-      String organization,
-      String areaOfResearch) {
+  public DbUser createUser(String givenName, String familyName, String userName) {
     return createUser(
```

Comment:
I'm declaring shenanigans on this constructor. We need a builder. 

There's no reason in Java 8+ I should ever have to see the word `null`, let alone 9 in a row. But that may be another ticket. 

If you don't want a builder, then your shorter constructors need to call the longer ones with default methods.

```
User createUser(A a) {
  return createUser(a, defaultB);
}

User createUser(A a, B b) {
  return createUser(a, b, defaultC);
}
// etc 
```
But a builder is still much preferred.

Reply:
This is a good idea, but a larger refactoring than I am comfortable doing right now